<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RunCmdSettingsDlg</class>
 <widget class="QDialog" name="RunCmdSettingsDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>587</width>
    <height>304</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Run Command - Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Select a command to be run when the experiment is started.&lt;/p&gt;&lt;p&gt;The command will have the following environment variables available to get more information about the experiment:&lt;/p&gt;&lt;p&gt;* SY_START_TIME_UNIX_MS - UNIX timestamp of the experiment start, in milliseconds&lt;/p&gt;&lt;p&gt;* SY_SUBJECT_ID - Identifier of the test subject&lt;/p&gt;&lt;p&gt;* SY_SUBJECT_GROUP - Group of the test subject&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="horizontalSpacing">
      <number>6</number>
     </property>
     <property name="verticalSpacing">
      <number>6</number>
     </property>
     <property name="topMargin">
      <number>6</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="exeLabel">
       <property name="text">
        <string>Executable</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="exeLineEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="parametersLabel">
       <property name="text">
        <string>Parameters</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="parametersLineEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="runOnHostLabel">
       <property name="text">
        <string>Run on host</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QCheckBox" name="runOnHostCheckBox">
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>RunCmdSettingsDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>RunCmdSettingsDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
